### DISCUSSION TITLE
openbsd

### DISCUSSION THREAD

**Discussion by babut85** [pgj/freebsd-wifibox#51] [Q&A] [answered]
are there any plans to include openbsd in supported systems? in obsd things are much worse with wifi than in freebsd -_- i think that from a moral point of view cripples need to save first ;)

**Comment by pgj** [comment, marked as answer]
This solution is for FreeBSD.  Not because I have problems with OpenBSD, I simply do not use it therefore I cannot provide support for it.  In addition to that, if I understand correctly, OpenBSD's hypervisor does not support hardware pass-through which is what the whole idea revolves around.  However, I am glad to share my experiences if anybody decides to port this solution to OpenBSD.
